README.md

```diff
@@ -1,2 +1,135 @@
+
+Events are generated when this service detects a change in Delius via the Delius OFFENDER_DELTA table. Rows are added via Delius database triggers. These triggers are currently limited changes to core offender changes; e.g changes to the management of a sentence will not typically raise an event.
+
+For each change detected two events will be raised:
+
+*   a generic event **OFFENDER_CHANGED**
+*   a more specific event with greater detail on what has changed: e.g **OFFENDER_ALIAS_CHANGED**
+
+We recommend subscribing to the specific events, the **OFFENDER_CHANGED** is deprecated and kept only for backward compatibility with existing subscribers.
+
+The specific events currently being raised are
+
+*   **OFFENDER_ADDRESS_CHANGED** is raised when an offender's address is changed
+*   **OFFENDER_DETAILS_CHANGED** is raised when offender's details has changed, e.g. date of birth
+*   **OFFENDER_MANAGER_CHANGED** is raised when an offender manager is allocated to the offender
+*   **OFFENDER_ALIAS_CHANGED** is raised when a alias is changed
+*   **OFFENDER_OFFICER_CHANGED** is raised when a current offender manager name changes
+
+### Topic subscription
+
+Clients are expected to use a SQS AWS queue to receive events with queue subscribed to **probation-events-topic**.
+
+Clients can subscribe to one or more events. A typical subscription could be:
+
+<pre>    resource "aws_sns_topic_subscription" "my_probation_subscription" {
+    provider      = aws.london
+    topic_arn     = module.probation_offender_events.topic_arn
+    protocol      = "sqs"
+    endpoint      = module.my_queue.sqs_arn
+    filter_policy = "{\"eventType\":[ \"OFFENDER_DETAILS_CHANGED\", \"OFFENDER_ADDRESS_CHANGED\"] }"
+    }
+
+</pre>
+
+The DPS Tech Team will create the queue and subscription on your behalf. If you are using the MoJ Cloud Platform we will need your namespace to send secrets there for your new queue.
+
+## Architecture and design
+
+The service uses a Spring Scheduler to pull the OFFENDER_DELTA table via the **community-api** 
+
+The scheduler will request all records every 10 seconds and process all records found. Multiple instances of this service can run currently since each record is "locked" when retrieved.
+The lock is "released" by either deleting the record or marking it as permanently failed. Any records that have not been processed with 10 minutes is assumed to ready to be processed again. This logic is encapsulated in the **community-api**.
+
+### Running
+
+`localstack` is used to emulate the AWS SQS and Elastic Search service. When running the integration test this will be started automatically. If you want the tests to use an already running version of `localstack` run the tests with the environment `AWS_PROVIDER=localstack`. This has the benefit of running the test quicker without the overhead of starting the `localstack` container.
```

Comment:
I'm not sure this is right - we're not using testcontainers in this project so localstack needs to be started manually before running the integration tests.

Reply:
Yep, removed this section